test/roby_app/ftp_server/test_server.rb

```diff
@@ -41,42 +42,44 @@ def upload_log(host, port, certificate, user, password, localfile)
         end
     end
 
-    # tests
-
-    def test_connects_to_server
-        spawn_server
-        Net::FTP.open("127.0.0.1", port: @server.port, verify_mode: OpenSSL::SSL::VERIFY_PEER, ca_file: @certificate) do |ftp|
-            assert ftp.login("test.user", ""), "FTP server doesn't connect."
+    ### TESTS ###
+    describe "#LogTransferServerTests" do
+        
+        it "tests connection to server" do
+            spawn_server
+            Net::FTP.open("127.0.0.1", port: @server.port, verify_mode: OpenSSL::SSL::VERIFY_PEER, ca_file: @certificate) do |ftp|
+                assert ftp.login("test.user", ""), "FTP server doesn't connect."
+            end
         end
-    end
 
-    def test_password_authentication
-        spawn_server_with_password
-        Net::FTP.open("127.0.0.1", port: @server.port, verify_mode: OpenSSL::SSL::VERIFY_PEER, ca_file: @certificate) do |ftp|
-            assert ftp.login("test.user", "password123"), "FTP server doesn't connect with authentication."
+        it "tests password authentication" do
+            spawn_server_with_password
+            Net::FTP.open("127.0.0.1", port: @server.port, verify_mode: OpenSSL::SSL::VERIFY_PEER, ca_file: @certificate) do |ftp|
+                assert ftp.login("test.user", "password123"), "FTP server doesn't connect with authentication."
+            end
         end
-    end
 
-    def test_uploads_file_to_server
-        spawn_and_upload_testfile
-        File.delete("testfile") if File.exist?("testfile") # delete in local directory
-        assert File.exist?("#{@temp_dir}/testfile"), "Uploaded file doesn't exist."
-    end
+        it "tests file uploads to server" do
+            spawn_and_upload_testfile
+            File.delete("testfile") if File.exist?("testfile") # delete in local directory
```

Comment:
This is a test. You should know whether testfile should or should not be present.